docs/GovernanceModel.md

```diff
@@ -41,20 +41,18 @@ Remember that security issues should be reported through a separate channel, and
 will receive a response within 24 hours. See [Reporting Security
 Issues](Contributing.md#reporting-security-issues).
 
-Maintainers, Committers, and Contributors
------------------------------------------
+Community Governance Committee Members, Committers, and Contributors
+--------------------------------------------------------------------
 
-We define "maintainer" as members of the Open Enclave "Community Maintenance
-Committee", as listed in the [maintainers document](Maintainers.md). A
-"committer" is anyone with direct write access to the Open Enclave repository on
-GitHub, as granted by the Committee. All maintainers are committers, but not all
-committers are maintainers. Finally, "contributor" is anyone else making
+A "committer" is anyone with direct write access to the Open Enclave repository on
+GitHub, as granted by the Committee. All Committee members are committers, but not all
+committers are Committee members. Finally, "contributor" is anyone else making
 contributions to the project, including: creating or commenting on issues,
 opening or reviewing pull requests, or other useful contributions such as
 providing support in forums or chats.
 
-See the [maintainers document](Maintainers.md) for the Community Maintenance
-Committee, our process for adding new committers and maintainers, as well the
+See the [Community Governance Committee document](Maintainers.md) for more information
```

Comment:
Should the document 'Maintainers.md' itself be renamed to 'Commiters.md'?

Reply:
I've been wondering that myself. They're both fairly common file names; I'd be fine with "Committers.md"

Reply:
Eventually yes, but I wasn't sure whether the GitHub diffs would make it hard to review so was going to do it in a subsequent PR.   (If everyone tells me it'll be fine, I can do it in this one.)

Reply:
If the concern is that history is not preserved because Git won't track a rename and edit correctly in a single commit, consider just adding a second rename commit to this PR.